content/en/tracing/trace_collection/trace_context_propagation/ruby.md

```diff
@@ -12,15 +12,15 @@ further_reading:
 
 ### B3 headers extraction and injection
 
-Datadog APM tracer supports [B3][6] and [W3C (TraceParent)][7] header extraction and injection for distributed tracing.
+Datadog APM tracer supports [B3][6] and [W3C Trace Context][7] header extraction and injection for distributed tracing.
 
 Distributed headers injection and extraction is controlled by configuring injection and extraction styles. The following styles are supported:
 
-- `Datadog`: Datadog style headers
-- `b3multi`: B3 multiple-headers
-- `b3`: B3 single-header
-- `tracecontext`: W3C Trace Context
-- `none`: No-op
+- Datadog: `Datadog`
+- B3 Multi Header: `b3multi`
+- B3 Single Header: `b3`
+- W3C Trace Context: `tracecontext`
+- No-op: `none`
```

Comment:
(Should we add `none` to dotnet? It works because it doesn't match any of the supported type, and we know a few users are using it to disable propagation, but it's not documented.)